cpp/farm_ng/core/enum/enum.h

```diff
@@ -25,14 +25,23 @@
 
 #include "farm_ng/core/enum/enum_without_ostream.h"
 
+#include <istream>
 #include <ostream>
 
-#define FARM_ENUM_OSTREAM_OVERLOAD(EnumName)                                \
-  namespace enum_wrapper_ {                                                 \
-  inline std::ostream &operator<<(std::ostream &os, EnumName##Impl value) { \
-    os << toPretty(value);                                                  \
-    return os;                                                              \
-  }                                                                         \
+#define FARM_ENUM_OSTREAM_OVERLOAD(EnumName)                                 \
```

Comment:
Maybe rename to FARM_ENUM_IOSTREAM_OVERLOAD?

Reply:
Good idea! Thanks Ian!